package.json

```diff
@@ -27,7 +27,10 @@
     "prettier": "prettier .",
     "clean-fast": "for dir in node_modules cache cache-zk artifacts artifacts-zk dist typechain; do mv \"${dir}\" \"_${dir}\"; rm -rf \"_${dir}\" &; done",
     "clean": "rm -rf node_modules cache cache-zk artifacts artifacts-zk dist typechain",
-    "build": "hardhat compile && anchor build && tsc && rsync -a --include '*/' --include '*.d.ts' --exclude '*' ./typechain ./dist/",
+    "build-evm": "hardhat compile",
+    "build-svm": "echo 'Generating IDLs...' && anchor build > /dev/null 2>&1 || true && anchor run generateExternalTypes && anchor build",
```

Comment:
note this `build-svm` command right now is super hacky. in order for `anchor build` to run correctly first time we require that first `anchor run generateEternalTypes` is run. however, this second command requires the IDLs and types to be generated, which are first built from `anchor build`. i.e we run the build command, which generates the IDLs and types, but then fails to build, which is then used in the `generateExternalTypes` command, which is then used again in the anchor build. 

we will improve this in a follow on PR but for now it works as expected, despite being a bit round about.